src/main/java/racing/domain/RacingHistory.java

```diff
@@ -0,0 +1,26 @@
+package racing.domain;
+
+import java.util.Set;
+
+public class RacingHistory implements Comparable<RacingHistory>{
+    private final Integer round;
+    private final Set<Car> HistorySet;
```

Comment:
```suggestion
    private final List<Car> HistoryList;
```

타입이 변경되면 변수이름도 변경되어야할까요?
일반화된 이름으로 표현하는건 어떨까요?